sdk/sdk/src/main/java/org/apache/carbondata/sdk/file/CarbonWriterBuilder.java

```diff
@@ -224,7 +226,8 @@ public CarbonWriterBuilder withLoadOptions(Map<String, String> options) {
           !option.equalsIgnoreCase("quotechar") &&
           !option.equalsIgnoreCase("escapechar") &&
           !option.equalsIgnoreCase("binary_decoder") &&
-          !option.equalsIgnoreCase("fileheader")) {
+          !option.equalsIgnoreCase("fileheader") &&
+          !option.equalsIgnoreCase("line_separator")) {
```

Comment:
can you please add one test case for sdk ?

Reply:
> can you please add one test case for sdk ?

Since line separator has no usage in sdk, remove the changes.